src/connectors/panoptic/panoptic.controllers.ts

```diff
@@ -685,52 +785,61 @@ export async function numberOfPositions(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: NumberOfPositionsRequest
-): Promise<string> {
+): Promise<NumberOfPositionsResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
   const numOfPositions = await panopticish.numberOfPositions(
     wallet
   );
   // TODO: cast this to number/int
-  return numOfPositions;
+  return {
+    numberOfPositions: numOfPositions
+  }
 }
 
 export async function optionPositionBalance(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: OptionPositionBalanceRequest
-): Promise<string> {
+): Promise<OptionsPositionBalanceResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
   const result = await panopticish.optionPositionBalance(
     wallet,
     req.tokenId
   );
-  // TODO: cast this to number/int
-  return result;
+  return {
+    balance: result.balance,
+    poolUtilization0: result.poolUtilization0, 
+    poolUtilization1: result.poolUtilization1
+  };
 }
 
 export async function pokeMedian(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: PokeMedianRequest
-): Promise<string> {
+): Promise<PokeMedianResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
-  const txData = await panopticish.pokeMedian(wallet);
-  return txData['hash'];
+  const txData = await panopticish.pokeMedian(wallet); 
+  return {
+    other: txData
+  }
 }
 
 export async function settleLongPremium(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: SettleLongPremiumRequest
-): Promise<string> {
+): Promise<SettleLongPremiumResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
   const positions = await panopticish.settleLongPremium(
     wallet,
     req.positionIdList,
     req.owner,
     req.legIndex
   );
-  return positions;
+  return {
+    other: positions
```

Comment:
Let's rename this field to `positions`, if these are indeed `positions` - but i'm surprised the `settleLongPremium` call returns that

Reply:
The ABI suggests it doesn't return anything: `{
      "inputs": [
        {
          "internalType": "TokenId[]",
          "name": "positionIdList",
          "type": "uint256[]"
        },
        {
          "internalType": "address",
          "name": "owner",
          "type": "address"
        },
        {
          "internalType": "uint256",
          "name": "legIndex",
          "type": "uint256"
        }
      ],
      "name": "settleLongPremium",
      "outputs": [],
      "stateMutability": "nonpayable",
      "type": "function"
    }`

Reply:
Done: 239ca3678be64cf12a0baea45ae3275020d1a26f